js/data/report.json

```diff
@@ -534,5 +534,6 @@
     ]
   },
   "anonymous_report": false,
-  "is_offering_external": true
+  "is_offering_external": true,
+  "hideControls": false
```

Comment:
I guess we'll receive this prop from Portal as `hide_controls`, so I would make it consistent, as this file is somewhat documenting expected API too.